.travis.yml

```diff
@@ -10,91 +10,119 @@ before_install:
   - sleep 15
   - sudo apt-get install -y
   - sudo apt-get install -y curl
+  - sudo apt-get install -y shellcheck
+
+language: shell
+  script:
+    - bash -c 'shopt -s globstar nullglob; shellcheck **/*.{sh,ksh,bash}'
+    - shellcheck busybox/liveness.sh custom-percona/docker-entrypoint.sh custom-percona/sql-test.sh dd-client/io_runner.sh debug-scripts/openebs/get_all_logs.sh elasticsearch-stress/elasticsearch-stress-test/run.sh fio/fio_runner.sh gitlab-runner/buildscripts/build.sh libiscsi/testiscsi.sh logger/logger.sh mongo-client/sysbench-mongo/sysbench/autogen.sh mongo-client/sysbench-mongo/sysbench/install-sh mysql-client/mysql-liveness-check.sh mysql-client/MySQLLoadGenerate.sh mysql-master/docker-entrypoint.sh mysql-slave/docker-entrypoint.sh postgres-client/workloads/test.sh sysbench/src/sysbench-runner.sh tpcc-client/tpcc-mysql/tpcc-runner.sh tpcc-client/tpcc-mysql/load.sh tpcc-client/tpcc-mysql/load_multi_schema.sh tpcc-client/tpcc-mysql/scripts/analyzeR.sh tpcc-client/tpcc-mysql/scripts/runX.sh tpcc-client/tpcc-mysql/scripts/2instances/run.sh tpcc-client/tpcc-mysql/scripts/innodb_stat.sh tpcc-client/tpcc-mysql/scripts/analyze_flushed.sh tpcc-client/tpcc-mysql/scripts/run.sh tpcc-client/tpcc-mysql/scripts/analyze_evicted.sh tpcc-client/tpcc-mysql/scripts/remote/runX.sh tpcc-client/tpcc-mysql/scripts/run_no_backup.sh tpcc-client/tpcc-mysql/scripts/analyze_checkpoint_xtradb.sh tpcc-client/tpcc-mysql/scripts/analyze_min.sh tpcc-client/tpcc-mysql/scripts/analyze_dirty.sh tpcc-client/tpcc-mysql/scripts/virident_stat.sh tpcc-client/tpcc-mysql/scripts/analyze_checkpoint.sh tpcc-client/tpcc-mysql/scripts/multi-instances/runX.sh tpcc-client/tpcc-mysql/scripts/analyze_checkpoint_xtradb1.sh tpcc-client/tpcc-mysql/scripts/analyze_modified.sh tpcc-client/tpcc-mysql/scripts/analyze_flushed_innodb.sh tpcc-client/tpcc-mysql/scripts/flashcache_stat.sh tpcc-client/tpcc-mysql/scripts/analyze.full.sh tpcc-client/tpcc-mysql/scripts/analyze.sh vdbench/vdbench.bash vdbench/bench_runner.sh
```

Comment:
Can we obtain a list of bash scripts via a bash command and iterate over that instead of listing them manually here?

Reply:
Okay Sure! I'll do it